src/control/log/log.go

```diff
@@ -87,14 +85,6 @@ func SetOutput(w io.Writer) {
 	logger.logger.SetOutput(w)
 }
 
-// WrapAndLogErr is a utility function that wraps and logs an error to the
```

Comment:
Why are we removing this and what is it replaced with? I'm using it in one of my PRs so if we're going to remove it I should clean up my usage before it merges.

Reply:
in discussions with @liw we concluded that it was better to do this explicitly given concerns around errors package and Wrap() that he sent an e-mail about

Reply:
Yes. I proposed to follow what the author of the errors package suggests, with only a minor amendment: write our own helper function to convert an error to a string, for logging, as the "%+v" implementation is a bit hard to read.